spec/controllers/posts_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
+    before(:each) do
+      @user = User.new(name: 'Tom', postscounter: 0)
+      @user.save
+      get '/users/1/posts'
+    end
+
+    it 'http request is successfull' do
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders correct' do
+      expect(response).to render_template(:index)
+    end
+  end
+end
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
+    before(:each) do
+      @user = User.new(name: 'Tom', postscounter: 0)
+      @post = Post.new(title: 'Title', text: 'text test', commentscounter: 1, likescounter: 0, user_id: 1)
+      @user.save
+      @post.save
+      get '/users/1/posts/2'
+    end
+    it 'renders correct template' do
```

Comment:
* Kindly add tests for your show actions as stated in the [requirements ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=Make%20sure%20that%20for,includes%20correct%20placeholder%20text.)
* Also i get these errors when i run the tests, Kindly fix them 
![image](https://user-images.githubusercontent.com/52497246/185089450-614cbbaa-1eaa-4d0a-a31c-4fd7dd42c9e9.png)

Reply:
Hello @Davidosky007 all the tests are passing on my local machine what could be the issue can you please elaborate on that

Reply:
When i run `rspec` i get those errors from my end, i will suggest that you update your readme file with steps on how you were able to achieve this on your end